frontend/src/metabase/sharing/components/SharingSidebar.jsx

```diff
@@ -471,8 +472,8 @@ class SharingSidebar extends React.Component {
             {({ onClose }) => (
               <DeleteModalWithConfirm
                 objectType="pulse"
-                title={t`Archive` + ' "' + pulse.name + '"?'}
-                buttonText={t`Archive`}
+                title={jt`Delete this subscription to ${pulse.name}?`}
```

Comment:
```suggestion
                title={t`Delete this subscription to ${pulse.name}?`}
```
Sorry I led you astray, @paulrosenzweig commented that only `t` is needed here. I'll fix this here. Otherwise looks good.

Reply:
Ah gotcha. Thanks!